sysutils/yabai/files/patch-makefile-macos11.diff

```diff
@@ -0,0 +1,40 @@
+--- makefile.orig	2023-10-10 14:06:45
++++ makefile	2023-10-17 17:18:17
+@@ -1,6 +1,6 @@
+ FRAMEWORK_PATH = -F/System/Library/PrivateFrameworks
+ FRAMEWORK      = -framework Carbon -framework Cocoa -framework CoreServices -framework SkyLight
+-BUILD_FLAGS    = -std=c99 -Wall -g -O0 -fvisibility=hidden -mmacosx-version-min=11.0 -fno-objc-arc -arch x86_64 -arch arm64
++CFLAGS         += -std=c99 -Wall -DNDEBUG -fvisibility=hidden -fno-objc-arc -arch x86_64
+ BUILD_PATH     = ./bin
+ DOC_PATH       = ./doc
+ SCRIPT_PATH    = ./scripts
+@@ -16,18 +16,18 @@
+ 
+ all: clean-build $(BINS)
+ 
+-asan: BUILD_FLAGS=-std=c99 -Wall -g -O0 -fvisibility=hidden -fsanitize=address,undefined -mmacosx-version-min=11.0 -fno-objc-arc -arch x86_64 -arch arm64
++asan: BUILD_FLAGS=-std=c99 -Wall -g -O0 -fvisibility=hidden -fsanitize=address,undefined -mmacosx-version-min=11.0 -fno-objc-arc -arch x86_64
+ asan: clean-build $(BINS)
+ 
+-tsan: BUILD_FLAGS=-std=c99 -Wall -g -O0 -fvisibility=hidden -fsanitize=thread,undefined -mmacosx-version-min=11.0 -fno-objc-arc -arch x86_64 -arch arm64
++tsan: BUILD_FLAGS=-std=c99 -Wall -g -O0 -fvisibility=hidden -fsanitize=thread,undefined -mmacosx-version-min=11.0 -fno-objc-arc -arch x86_64
+ tsan: clean-build $(BINS)
```

Comment:
I'm surprised these lines need patching, as we don't (as far as I can tell) call `make asan` or `make tsan` in the build. If we can get away with it, patching fewer things would probably be helpful in keep this easier to maintain. Of course, I could also have missed how this gets used!

If this is indeed being used, we may also need to make the `s/BUILD_FLAGS/CFLAGS/` switch here, to make sure things remain compatible with any user CFLAGS configurations.

Reply:
My bad, I just deleted each instance of "arm64" from the original makefile. I've removed these lines, since it doesn't seem they require patching.